ctapipe/reco/reco_algorithms.py

```diff
@@ -93,3 +94,18 @@ def _get_telescope_pointings(event):
             )
             for tel_id in event.dl1.tel.keys()
         }
+
+    def _store_impact_parameter(self, event):
+        """Compute and store the impact parameter for each reconstruction."""
+        impact_distances = shower_impact_distance(
+            shower_geom=event.dl2.stereo.geometry[self.__class__.__name__],
+            subarray=self.subarray,
+        )
+        for tel_id in event.trigger.tels_with_trigger:
+            tel_index = self.subarray.tel_indices[tel_id]
+            event.dl2.tel[tel_id].impact[
+                self.__class__.__name__
+            ] = TelescopeImpactParameterContainer(
+                distance=impact_distances[tel_index],
+                prefix=f"{self.__class__.__name__}_tel",
```

Comment:
Here the default prefix should be added, see https://github.com/cta-observatory/ctapipe/pull/2060